lms/static/scripts/frontend_apps/components/test/GradingControls-test.js

```diff
@@ -139,7 +139,7 @@ describe('GradingControls', () => {
         displayName: 'Student Beta',
       },
       {
-        displayName: undefined,
+        displayName: '',
```

Comment:
Our types define `displayName` to be a string and as far as I can see, the value is always initialized as such on the backend.